adhocracy-plus/assets/scss/_variables.scss

```diff
@@ -81,6 +81,7 @@ $mobile-headline-size: 0.8 !default;
 
 $shadow: rgba(#000, 0.25) !default;
 
+$em-spacer: 1em !default;
 $spacer: 1rem !default;
```

Comment:
we try not to use em spacing where possible, any that you've found is legacy, is there a reason this is needed?